packages/coq-of-ocaml/coq-of-ocaml.2.5.3+4.12/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "dev@clarus.me"
+homepage: "https://github.com/formal-land/coq-of-ocaml"
+dev-repo: "git+https://github.com/formal-land/coq-of-ocaml.git"
+bug-reports: "https://github.com/formal-land/coq-of-ocaml/issues"
+authors: ["Guillaume Claret"]
+license: "MIT"
+build: [
+  ["sh" "-c" "cd proofs && ./configure.sh"] {coq:installed}
+  [make "-C" "proofs" "-j%{jobs}%"] {coq:installed}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+install: [
+  [make "-C" "proofs" "install"] {coq:installed}
+]
+depends: [
+  "angstrom"
```

Comment:
You are missing a lower-bound for this constraint. This is causing the CI error:

```
# File "src/constant.ml", line 55, characters 13-25:
# 55 | let dquote = Angstrom.map ~f:(fun _ -> PDQuote) (Angstrom.char '"')
#                   ^^^^^^^^^^^^
# Error: Unbound value Angstrom.map
```

I think the `map` function was introduced in 0.15.0: https://github.com/inhabitedtype/angstrom/commit/93f29c2343491ef773efc89d2fa4e695a5bd1202

So I'd suggest

```suggestion
  "angstrom" {>= "0.15.0"}
```

If this is correct, then you should also upstream this change.

Reply:
Note: the same change is needed on all three package files.

Reply:
Thanks! I have added the constraint for all three packages.